google_compute_engine/accounts/tests/accounts_utils_test.py

```diff
@@ -148,6 +148,27 @@ def testCreateSudoersGroupError(
     ]
     self.assertEqual(mocks.mock_calls, expected_calls)
 
+  @mock.patch('google_compute_engine.accounts.accounts_utils.open')
```

Comment:
Example of mock_open:
https://github.com/GoogleCloudPlatform/compute-image-packages/blob/master/google_compute_engine/accounts/tests/accounts_utils_test.py#L76